library/src/main/res/values/dimens.xml

```diff
@@ -3,6 +3,8 @@
   <dimen name="calendar_day_headers_paddingbottom">4dp</dimen>
   <dimen name="calendar_month_topmargin">16dp</dimen>
   <dimen name="calendar_month_title_bottommargin">4dp</dimen>
+  <dimen name="calendar_month_title_paddingtop">16dp</dimen>
```

Comment:
I think you also need to remove these and change their uses over to the old dimens